selftests/check.py

```diff
@@ -632,6 +631,10 @@ def create_suites(args):  # pylint: disable=W0621
             {
                 "runner": "avocado-runner-pip",
             },
+            {
+                "runner": "avocado-runner-vmimage",
+                "extra_config": {"run.max_parallel_tasks": 1},
```

Comment:
IMO this is a left over from development, and it is not needed for CI. The runner should be able to work in parallel execution, am I right?